### PR TITLE
Add `markup.` scopes in `gruvbox` themes

### PR DESCRIPTION
**Pull request by crodjer**
As recommended by @archseer in https://github.com/helix-editor/helix/pull/1509#issuecomment-1013583069

### CODE REVIEW

File: runtime/themes/gruvbox_light.toml [hunk 1]
```diff
@@ -20,6 +20,7 @@
 "function" = { fg = "green1", modifiers = ["bold"] }
 "function.macro" = "aqua1"
 "function.builtin" = "yellow1"
+"tag" = "red1"
```
Comment (pickfire) [inline runtime/themes/gruvbox_light.toml:23]: Why is tag in gruvbox light but not in gruvbox?

**Comment by archseer** [comment]
Thanks for adding these!

File: runtime/themes/gruvbox_light.toml (hunk 1 above)
Comment (crodjer) [reply runtime/themes/gruvbox_light.toml:23]: Ahh, missed this. Will send a patch. Thanks for pointing it out 🙇🏽

Comment (crodjer) [reply runtime/themes/gruvbox_light.toml:23]: Done @pickfire: https://github.com/helix-editor/helix/pull/1555

**Pull request merged**